src/eddsa.rs

```diff
@@ -45,7 +45,7 @@ impl SecretKey {
 
 /// `PublicKey` is EdDSA signature verification key
 #[derive(Copy, Clone, Debug, CanonicalSerialize, CanonicalDeserialize)]
-pub struct PublicKey<TE: TECurveConfig>(Affine<TE>);
+pub struct PublicKey<TE: TECurveConfig>(pub Affine<TE>);
```

Comment:
Could use existing `as_ref` method to reach inner

Reply:
cool, removed the `pub` and used the `as_ref` method at https://github.com/kilic/arkeddsa/pull/1/commits/8efc724124d1dbf550a7d00d12ff3920851bedf2